neasqc_wp61/models/quantum/embedder.py

```diff
@@ -0,0 +1,322 @@
+"""
+Embedder
+============
+Module containing the base class for transforming sentences in a dataset into embeddings
+"""
+
+from abc import ABC, abstractmethod
+
+import pandas as pd
+from pandas.core.api import DataFrame as DataFrame
+from transformers import BertTokenizer, BertModel
+import torch
+import numpy as np
+import fasttext as ft
+import fasttext.util as ftu
+
+
+class Embedder(ABC):
+    """
+    Base class for the generation of embeddings for the sentences
+    comprising a dataset.
+    """
+
+    def __init__(self, dataset: DataFrame, sentence_or_word: bool = True) -> None:
```

Comment:
Pls change the name of this variable. sentence_or_word = True ==> what does that even mean?
If you're going for sentence VS word pls choose one of the other as being True and name the variable accordingly as in for instance:
sentence_embedding = True #gives sentence embeddings
sentence_embeddings = False #gives word embeddings

Reply:
addressed

Reply:
reason why I went with that name is because it was in the UML diagram, but I agree with you!

Reply:
Sorry,,gave a name just for illustration, have made a new comment on why it should be slightly different.

Reply:
addressed (this is for me to keep track of addressed and unaddressed comments)